README.md

```diff
@@ -60,7 +60,7 @@ running the following command:
     rails generate delayed_job:active_record
     rake db:migrate
 
-For Rails 4.2, see [below](#rails-42)
```

Comment:
I was about to PR this too. I think changing 'Rails 4.2' -> 'Rails 4.2+' would also be more accurate.